liquibase-standard/src/main/java/liquibase/GlobalConfiguration.java

```diff
@@ -221,6 +224,11 @@ public class GlobalConfiguration implements AutoloadedConfigurations {
                 .setDescription("Will perform xsd validation of XML changelog files. When many XML changelog files are included this validation may impact Liquibase performance. Defaults to true.")
                 .setDefaultValue(true)
                 .build();
+
+        UI_SERVICE = builder.define("uiService", UIServiceEnum.class)
+                .setDescription("Changes the default UI Service Logger used by Liquibase. Option are CONSOLE or LOGGER - defaults to CONSOLE.")
```

Comment:
```suggestion
                .setDescription("Changes the default UI Service Logger used by Liquibase. Options are CONSOLE or LOGGER.")
```

Reply:
The default value automatically shows up in the console output, no need to repeat it in the description.